docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -86,6 +88,7 @@ Example:
 |--------	|--------------|:-------:|:--------:|
 | `namespace` | Namespace to deploy the Prometheus Operator. | - | true |
 | `grafana_admin_password` | Password for `admin` user in Grafana.  | - | true |
+| `grafana_ingress_host` | Ingress URL Host to expose Grafana over internet. **NOTE:** User will have to create a DNS entry for this against Packet EIP manually if running on Packet.  | - | false |
```

Comment:
>**NOTE:** User will have to create a DNS entry for this against Packet EIP manually if running on Packet.

It feels weird to me to include a note in the knob's description. Is there a better place to put this info?

I suggest the following alternative phrasing: "When running on Packet, a DNS entry pointing at the ingress controller needs to be created." I think it's better because of the following:

- "User will have to..." is IMO the wrong tone for using in documentation. We either use the imperative tense "Do this and that", or we use the passive tense "This and that needs to be done".
- "Packet EIP" is ambiguous and assumes context - we assume the user knows what an EIP is and that we are talking specifically about the EIP used for Contour. We should be more concrete.
- There is a missing article ("the") before "Packet WIP".
- The condition ("if running on Packet") should come before the instruction to avoid forcing the user to read a block of text only to discover it doesn't apply to their situation when reaching its end ("I'm running on AWS").

Reply:
> If we create a wildcard record pointing at Contour on AWS, why don't we do it on all platforms?

This will solve the problem for Packet yes. And it will make things easier for the user on Packet.

Reply:
Created an issue for this https://github.com/kinvolk/lokomotive/issues/470

Reply:
Made changes as you have suggested.